src/FacadeAccountCreation/FacadeAccountCreation.API/appsettings.json

```diff
@@ -58,6 +58,9 @@
   "OrganisationEndpoints": {
     "UpdateOrganisation": "api/organisations/organisation"
   },
+  "UserDetailsEndpoints": {
+    "UpdateUserDetails": "api/persons/update-user-details?userId="
```

Comment:
Personally i would make this RESTful (i.e. api/persons/update-user-details/[user-id])